components/TransactionItem/TransactionItem.module.css

```diff
@@ -0,0 +1,65 @@
+.list-item {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding-left: 2%;
+    padding-right: 2%;
+    height: 62px;
+    border-top: 1px solid #EBEBEB;
+  }
+
+.date {
+    width: 8%;
+}
+
+.username {
+    width: 15%;
+}
+
+.fid {
+    width: 6%;
+}
+
+.admin {
+    width: 18%;
+}
+
+.redeem-action {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100px;
+    height: 30px;
+    background-color: #FFD4C7;
+    border: 1px solid #C8C8C8;
+    border-radius: 16.5px;
+}
+
+.earn-action {
```

Comment:
Just discussed this with @hukellyy, but could you make the redeem and earn chip/buttons the same width? I think it would look better on the UI.